src/Module.php

```diff
@@ -133,6 +133,26 @@ class Module extends \yii\base\Module implements BootstrapInterface
      */
     public $traceLine = self::DEFAULT_IDE_TRACELINE;
 
+     /**
+     * @var string|array|callable Page title could be a string, array or a callable function
+     * ```php
+     *  ...
+     *  'pageTitle'=> 'Custom Debug Title',
+     *  ...
+     * // OR
+     * 'pageTitle'=> [
+     *     'base-url'=> 'page title',
+     *     'http://qa.example.test'=> 'qa debugger'
+     * ]
+     * // OR
+     * 'pageTitle'=> function($url){
+     *     $domain = getDomain($url);
+     *     return $domain . ' debugger';
+     * }
+     * ```
```

Comment:
@m-wardany now that we have the callable form, do you think we can drop the array form to reduce complexity?

Reply:
I thought using array could be more easier if we don't have much urls,
but now I see that both will be the same